### ISSUE TITLE
Broken link to pre-built ROS image

### ISSUE THREAD

**Issue by ireasor** [nasa-jpl/open-source-rover#149]
In the Software Steps pdf, there is a link to a "Custom JPL image with both Raspbian Stretch and ROS pre-built" that points to https://drive.google.com/open?id=18Tzx2tzmwUHOpBue-Hew3XYpXZIEQpiF.  This URL returns a 404.

**Comment by ericjunkins** [comment]
prebuilt image can be found at the following for now: https://drive.google.com/drive/folders/1RbYWDRthpcktqkPEHb7qVqDiy27EiAc1

**Comment by ireasor** [comment]
Thanks @ejunkins.  I downloaded the image and have tried to load it onto an SD card with balenaEtcher.  Unfortunately, the image is 32.01GB and the SD card that I have is 31.91GB.  I've tried shrinking the disk image in the MacOS Disk Utility, but I haven't had any luck.  Is there any way on your end to make the image slightly (1-200MB) smaller?  Due to the variation in actual capacity between manufacturers, I expect that others will run into this issue as well.
